package.json

```diff
@@ -15,25 +15,25 @@
     "integration"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
```

Comment:
can we bump dev deps in a separate PR to make it smaller?

Reply:
Yup